temporalio/bridge/client.py

```diff
@@ -13,7 +13,6 @@
 
 import temporalio.bridge.runtime
 import temporalio.bridge.temporal_sdk_bridge
-from temporalio.bridge.temporal_sdk_bridge import RPCError
```

Comment:
This is imported for side effecting reasons as are some others in some other files. Feel free to find ways to workaround.